### PR TITLE
Add ability to print multiple stories at once

### PR DESCRIPTION
**Pull request by scottspeidel**
Hi Yi,

I am adding this pull request to add the ability to use the "story" command to print multiple stories at once with a single command. It would work as follows:

> pivotal_to_pdf story story_id1 story_id2 .... story_idn

I ran all the specs and they are passing. I do think there is room for improvement in the tests.  Any feedback is welcome and encouraged. I am happy to make modifications.

Thanks!
Scott

### CODE REVIEW

**Comment by scottspeidel** [comment]
I'll fix it.

**Comment by scottspeidel** [comment]
ugh... now it's broken from the command line. I WILL fix it this time. Ignore this PR for now.

**Comment by scottspeidel** [comment]
All green.

**Comment by ywen** [comment]
I will get back to you this weekend. Thanks for the effort

File: bin/pivotal_to_pdf [hunk 1]
```diff
@@ -4,9 +4,16 @@ require 'pivotal_to_pdf'
 require 'thor'
 
 class PivotalToPdfApp < Thor
-  desc "story STORY_ID", "print a single story specifed by ID into a PDF file. The card will have a color stripe. The color will be green for features, yellow for chores and red for bugs"
-  def story(story_id)
-    PivotalToPdf::Main.story story_id
+
+  desc "story [STORY_IDs]", "print one or more stories specifed by ID into a PDF file. The card will have a color stripe. The color will be green for features, yellow for chores and red for bugs"
+  def story(*story_ids)
```
Comment (ywen) [inline bin/pivotal_to_pdf]: Regarding your question on how to make it requiring at east one argument. I guess you could change the method signature to

```ruby
def story(story_id, *other_story_ids)
```

Comment (scottspeidel) [inline bin/pivotal_to_pdf]: Ok. That makes sense.

**Comment by ywen** [comment]
Other than that the change looks good to me

**Pull request merged**